ouroboros-consensus-cardano/src/unstable-cardano-tools/Cardano/Tools/DBAnalyser/Run.hs

```diff
@@ -70,6 +70,7 @@ analyse DBAnalyserConfig{analysis, confLimit, dbDir, selectDB, validation, verbo
                   (const True)
                   (Node.stdMkChainDbHasFS dbDir)
                   (Node.stdMkChainDbHasFS dbDir)
+                  (ApplyOpts ValidationOff OmitLedgerEvents)
```

Comment:
What's the relation between this `ValidationOff` and the `maybeValidateAll` wrapper?

Might deserve an inline comment clarifying.